360-Videos-Metadata/360VideosMetadata.py

```diff
@@ -768,11 +768,12 @@ def PrintMetadata(src):
 
     print "Processing: ", infile, "\n"
 
-    if ".mkv" in infile:
+    if (os.path.splitext(infile)[1].lower() == ".webm" or
+       os.path.splitext(infile)[1].lower() == ".mkv"):
```

Comment:
ah riiiighhhtt ... for future situations like that I'd indent next line by 4 from the (

(but still I think the "in [ ... ]" is the way to go here)